.github/workflows/root-ci-config/buildconfig/windows10.txt

```diff
@@ -13,7 +13,7 @@ builtin_xxhash=ON
 builtin_zlib=ON
 builtin_zstd=ON
 davix=OFF
-fftw3=OFF
+fftw3=ON
```

Comment:
Can't you just remove these three lines, since now all flags are consistent with `global.txt`?

Reply:
Indeed, thanks!

Reply:
Done. Thanks for the suggestion!